cypress/support/step_definition/course/createCourse/createCourse.spec.js

```diff
@@ -1,6 +1,12 @@
 import Create_Course from '../../../pages/course/pageCreateCourse'
+import Navigation from '../../../pages/common/pageNavigation'
 
 const createCourse = new Create_Course()
+const navigation = new Navigation()
+
+When ('I am in the rooms overview', () =>{
```

Comment:
Instead of adding this steps again to another spec file (it's also inside deleteCourse.spec.js), we should move it out. For example to a file like pageNavigation.spec.js, since this step will be used in a lot of testcases and counts as a common step.

Reply:
Agreed.

Reply:
1. The step definition should be aligned with features file w.r.t. all steps. so it should not be like we can skip 'When' from step definition but we have it in the feature file (looks not consistent)
2.  We already have this method 'goToRoomOverview()' inside page->common-> pageNaviagrtion.js, so we can call this method multiple time in different tests.

Reply:
But it's similar to the login step. We currently do it in all of our feature files. but don't mention it inside each spec file. Doing so would, imho, only produce redundant and not needed code.